src/chapters/http/http-in-the-browser.rst

```diff
@@ -0,0 +1,57 @@
+HTTP in the Browser
+===================
+
+While we have covered all of the HTTP concepts you need to know at this point, it's worth spending some time explaining how web browsers submit requests and receive responses.
+
+Viewing Requests and Response Using Developer Tools
```

Comment:
I'm not sure where to put it, what do you think about adding a link to the MDN dev tools page? https://developer.mozilla.org/en-US/docs/Tools

Reply:
I added it in a tip at the top of the section.